java/arcs/android/devtools/StoreMessage.kt

```diff
@@ -0,0 +1,163 @@
+package arcs.android.devtools
+
+import arcs.android.devtools.DevToolsMessage.Companion.STORE_MESSAGE
+import arcs.core.common.Referencable
+import arcs.core.crdt.CrdtData
+import arcs.core.crdt.CrdtOperation
+import arcs.core.crdt.CrdtSet
+import arcs.core.crdt.CrdtSingleton
+import arcs.core.crdt.VersionMap
+import arcs.core.data.RawEntity
+import arcs.core.data.util.ReferencablePrimitive
+import arcs.core.storage.ProxyMessage
+import arcs.core.util.JsonValue
+
+/**
+ * An implementation of [DevToolsMessage] to inform DevTools that a [Store] received a
+ * [CrdtOperation].
+ */
+class StoreMessage(
+    private val actualMessage: ProxyMessage.Operations<CrdtData, CrdtOperation, Any?>
+) : DevToolsMessage {
+    override val kind: String = STORE_MESSAGE
+    override val message: JsonValue<*>
+        get() = JsonValue.JsonObject(mapOf<String, JsonValue<*>>(
+            "id" to JsonValue.JsonNumber(actualMessage.id?.toDouble() ?: 0.0),
+            "operations" to JsonValue.JsonArray(getMessageAsList())
+        ))
```

Comment:
```suggestion
        get() = JsonValue.JsonObject(
            mapOf<String, JsonValue<*>>(
                "id" to JsonValue.JsonNumber(actualMessage.id?.toDouble() ?: 0.0),
                "operations" to JsonValue.JsonArray(getMessageAsList())
            )
        )
```

Reply:
Also looks like there are several places in this file where this type of nit needs to be picked (see: getMessageAsList, for example)

Reply:
Fixed!